eng/pipelines/coreclr/templates/run-test-job.yml

```diff
@@ -378,6 +378,13 @@ jobs:
           scenarios:
           - jitelthookenabled
           - jitelthookenabled_tiered
+        ${{ if in(parameters.testGroup, 'jit-offbydefault-extra') }}:
```

Comment:
Is there any alternatives for naming this pipeline other than `offbydefault`?

`jitexperimentalfeatures`? or just `jitexperimental`

My first thoughts when I read offbydefault were that this pipeline is off by default. But then when I saw a schedule trigger I realized that this is about the jit features.

Reply:
Happy to rename if it reduces confusion -- @dotnet/jit-contrib any other thoughts/suggestions?

Reply:
I like "jit-experimental". I don't think you need "-extra" (We could save that for a version of this that also does JitStress modes, perhaps.)